gn/BUILDCONFIG.gn

```diff
@@ -18,6 +18,10 @@ declare_args() {
   }
   linux_soname_version = ""
   sanitize = ""
+
+  building_for_tizen = false
```

Comment:
Could we replace this with a use of `target_os="tizen"` in the section below? Basically, like what we did with `winrt` and `tvos`.

Reply:
Done.